gap-analysis/index.md

```diff
@@ -19,18 +19,18 @@ layout: wgnote
 
-Since its beginnings in 1998, MathML has had two parts: Presentation MathML which describes the arrangement of math symbols, and Content MathML describes the composition of math operators.  These two subsets of MathML carry complementary information and can be used separately, or combined using Parallel Markup (described below).  Neither of these forms carries adequate information to provide unambiguous accessible output.  While some semantic information is explicit or can be inferred, other aspects of the semantics must be provided by an author to resolve ambiguities in conventional math notation to make the MathML encoding more fully accessible.  This document examines two central questions:
+Since its beginnings in 1998, MathML has had two parts: Presentation MathML which describes the arrangement of math symbols, and Content MathML describes the composition of math operators.  These two subsets of MathML carry complementary information and can be used separately, or combined using Parallel Markup (described below).  Neither of these forms carries adequate information to provide unambiguous accessible output. While some mathematical concepts are explicit or can be inferred, the general problem of understanding mathematical notation is too hard for present-day systems. For the hard cases, annotations need to be provided by an author. This document examines two central questions:
 
-* What additional semantic information is needed beyond current MathML and how should it be encoded?
+* What additional information is needed beyond current MathML and how should it be encoded?
 * How should that information be communicated to an end user to provide greater access to math content, especially for vocalization of math by Assistive Technology (AT) such as screen readers?
 
 This document makes no conclusions. It lists some of the strengths and weaknesses of the approaches presented. The reader is invited to comment on those approaches and/or provide alternatives as the WG wants to come to a solution that works within the web platform to address the problems noted below.
 
-The goal is to allow authors/authoring tools the ability to capture, in MathML, enough information so that an expression can be correctly rendered by screen reader applications. It is likely that additional information that aids accessibility can also be used to improve search of mathematical expressions, but the focus of this document is on enhancing accessibility.
+The goal is to allow authors/authoring tools the ability to capture, in MathML, enough information so that an expression can be correctly rendered by screen reader applications. It is likely that further applications may be enabled with the same annotations, but the focus of this document is on enhancing accessibility.
```

Comment:
Do you object to me adding something like
> ...further applications such as search, computation, and conversion to other formats may be enabled..
to
> ...further applications...

This seems to address Steve and Paul's concerns about eliminating some later sections.

Reply:
Sure, as long as it's weakened a little. `may be enabled` is a bit wishful thinking, I'd go prefer `may benefit from`.

Reply:
It's funny that I used `enabled` in the abstract without any issue, but the moment you name the applications I immediately back away from the word. Claim gets too real.